variables.tf

```diff
@@ -60,21 +60,36 @@ variable "vpc_id" {
   description = "VPC where the cluster and workers will be deployed."
 }
 
-variable "worker_groups" {
```

Comment:
Like I said before, can we make this PR backwards compatible? i.e leave this variable and all related stuff in place so that a user can use launch templates if they want but without destroying existing resources.

Reply:
Sorry, misinterpreted your previous ask there.
Will update PR